settings.gradle

```diff
@@ -31,6 +31,7 @@ include 'data-prepper-plugins:otel-trace-source'
 include 'data-prepper-plugins:peer-forwarder'
 include 'data-prepper-plugins:blocking-buffer'
 include 'data-prepper-plugins:http-source'
+include 'data-prepper-plugins:drop-processor'
```

Comment:
I think we should try to keep the names consistent. You named the plugin `drop_events`. So, I think this project should be `drop-events-processor`.

Reply:
Renamed to drop-events-processor